services/nginx/nginx.conf.template

```diff
@@ -5,6 +5,19 @@ pid run/nginx.pid;
 http {
   sendfile on;
 
+  geo $limit {
```

Comment:
This seems to mean "requests from localhost do not get rate limited".  Is that the intent?

Reply:
That's correct

Reply:
Without this, `make load_demo_data` fails

Reply:
Eventually we can template this to add Kowalski's IP address (in fritz-deploy)